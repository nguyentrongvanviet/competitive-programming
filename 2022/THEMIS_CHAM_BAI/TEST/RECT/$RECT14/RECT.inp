43
292794031 152354702
-517595275 -116999073
-714109167 801162212
-348519363 814966482
-592230418 -896761157
-554687811 -287051519
49014135 76283824
-950952609 233420255
-885962266 351870564
172172991 -58642721
113816639 631125545
895385885 -967794673
-709406921 5156642
271652770 -455444146
-417486372 131289330
-348519363 814966482
-881603253 -136437342
-633997695 867289406
-950952609 233420255
-575951627 572769193
-714109167 801162212
-546884897 274823570
-373034863 524970448
148829296 131852035
-709406921 5156642
897070107 -121059877
-968903036 363161252
-885962266 351870564
-750595019 333507468
372191281 566888596
49014135 76283824
-881603253 -136437342
-604117805 -382808058
372191281 566888596
-924158765 862318548
401713542 -248413460
-406733342 552078270
148829296 131852035
505866384 310380825
-262328328 778018122
-748788253 237807241
-748788253 237807241
-554687811 -287051519
