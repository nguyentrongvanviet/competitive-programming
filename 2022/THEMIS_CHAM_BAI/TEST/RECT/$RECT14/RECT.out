479185724599394660